### ISSUE TITLE
Review, finish, and document merge_noto.py

### ISSUE THREAD

**Issue by behdad** [googlei18n/noto-fonts#722]
We want to experiment with merging fonts into a few categories.  Need to revive the `merge_noto.py` script as a starter.

**Comment by jungshik** [comment]
Yeah... we played with various grouping ideas as well as how to handle overlapping code points.

**Comment by tzot** [comment]
Being quite fluent in Python, I'd like to help with this issue, but I don't know how to locate the `merge_noto.py` script.  
If I'm talking out of place, please feel free to ignore me. :)

**Comment by behdad** [comment]
Err, my bad.  I should have filed this against https://github.com/googlei18n/nototools

**Comment by hrhatada** [comment]
Hi Behdad, have you filed this request against https://github.com/googlei18n/nototools so that we can close this?

**Comment by behdad** [comment]
> Hi Behdad, have you filed this request against https://github.com/googlei18n/nototools so that we can close this?

No.  Feel free to do that.  Thanks.
